### ISSUE TITLE
Error: One of the glob patterns doesn't match any files

### ISSUE THREAD

**Issue by fabionolasco** [nuxt/nuxt.js#1292]
About 80% of the time when I run nuxt build or generate I get the error referred below. I get that error in brand new linux boxes in the cloud, as well as on my local Mac. The weird part is that sometimes, if I wait a few minutes and then run the command again, it works fine for no apparent reason.

You can try to replicate the error by cloning the DEV branch of one of my repos:

[https://github.com/fabionolasco/fabionolasco.com/tree/dev](https://github.com/fabionolasco/fabionolasco.com/tree/dev)

```
> nuxt build

  nuxt:build Building... +0ms
  nuxt:build App root: /fabio/fabionolasco.com +1s
  nuxt:build Generating /fabio/fabionolasco.com/.nuxt files... +0ms
  nuxt:build Generating files... +8ms
  nuxt:build Generating routes... +7ms
  nuxt:build Building files... +66ms
  build [==========          ] 50%Error: One of the glob patterns doesn't match any files. Please remove or fix the following: {
  "globDirectory": "/fabio/fabionolasco.com/.nuxt/dist",
  "globPattern": "**/*.{js,css}",
  "globIgnores": [
    "node_modules/**/*",
    "../../static/workbox-sw.prod.v1.1.0.js",
    "../../static/sw.js"
  ]
}
    at module.exports.err (/fabio/fabionolasco.com/node_modules/workbox-build/build/lib/utils/get-file-details.js:20:11)
    at globPatterns.reduce (/fabio/fabionolasco.com/node_modules/workbox-build/build/lib/get-file-manifest-entries.js:120:32)
    at Array.reduce (native)
    at getFileManifestEntries (/fabio/fabionolasco.com/node_modules/workbox-build/build/lib/get-file-manifest-entries.js:119:36)
    at copyWorkboxSW.then.then (/fabio/fabionolasco.com/node_modules/workbox-build/build/lib/generate-sw.js:150:12)
```

**Comment by fabionolasco** [comment]
Side note: The command "nuxt" by itself, to serve app as dev, works well.

**Comment by pi0** [comment]
Hi. This problem is with workbox module of pwa . Please temporary disable it using : 
```js
  modules: [
    [ '@nuxtjs/pwa', { workbox: false } ]
  ]
```

I'll investigate more about exact problem.

**Comment by fabionolasco** [comment]
Hi pi0 thank you so much! That did not work for me, but I will wait for the fix. I appreciate it. Btw, the error that I get when I implement your code is:

Error: [nuxt] Module {"workbox":false} should export a function
    at ModuleContainer._callee2$ (/usr/local/lib/node_modules/nuxt/lib/core/module.js:114:13)
    at tryCatch (/usr/local/lib/node_modules/nuxt/node_modules/regenerator-runtime/runtime.js:65:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/usr/local/lib/node_modules/nuxt/node_modules/regenerator-runtime/runtime.js:303:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/usr/local/lib/node_modules/nuxt/node_modules/regenerator-runtime/runtime.js:117:21)
    at step (/usr/local/lib/node_modules/nuxt/dist/nuxt.js:852:30)
    at /usr/local/lib/node_modules/nuxt/dist/nuxt.js:863:13
    at process._tickCallback (internal/process/next_tick.js:109:7)

**Comment by pi0** [comment]
Would you please check the version of Nuxt at `node_modules/nuxt/package.json` ? I think you are using an alpha version not rc.

**Comment by fabionolasco** [comment]
On my project's package.json is says: latest. And in the node_modules/nuxt/package.json it says   "version": "1.0.0-rc3".

**Comment by Darkside73** [comment]
Try to use this workaround in nuxt.config:
```js
'@nuxtjs/pwa', {
  workbox: {
    globPatterns: ['*'],
    cacheId: 'frontend_1.0.0'
  }
}]
```
This puts all files (including map, json) into SW though.
But it's eligible for testing purposes 
I've created an issue in modules repo https://github.com/nuxt-community/modules/issues/110

**Comment by pi0** [comment]
This should be fixed with @nuxtjs/workbox:^1.1.5 please upgrade :)

**Comment by fabionolasco** [comment]
Thank you @pi0
